libs/output-mapping/src/Writer/Table/TableDefinition.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\OutputMapping\Writer\Table;
+
+use Keboola\Datatype\Definition\Exasol;
+use Keboola\Datatype\Definition\Snowflake;
+use Keboola\Datatype\Definition\Synapse;
+
+class TableDefinition
+{
+    public const NATIVE_BACKEND_TYPE_CLASS_MAP = [
+        'snowflake' => Snowflake::class,
+        'synapse' => Synapse::class,
+        'exasol' => Exasol::class,
+    ];
+
+    public const BACKEND_COMPONENTS_MAP = [
```

Comment:
Toto se mě ani trošku nelibí, z toho se okotíme už při třetím updatu

Reply:
jasne, me taky ne

Reply:
let's avoid it then

Reply:
yeah, I'm just not quite sure how 😄 .  it's now been moved to the factory class